docs/testnet/onomy-testnet-docs/pre-installation.md

```diff
@@ -0,0 +1,35 @@
+
+In theory, Onomy chain can be run on Windows and Mac. Binaries will be provided on the releases page and currently, scripts files are provided to make binaries.
+I also suggest an open notepad or other document to keep track of the keys you will be generating.
+
+## Bootstrapping steps and commands
+
+Start by logging into your Linux server using ssh. The following commands are intended to be run on that machine
+
+### Download/install Onomy chain binaries
+To download binary follow these commands
+
+```
+cd $HOME
```

Comment:
instead of making their own binary, the binary link is provided to download and move on.
If users want to make their own binary all these steps all already written in the script file.